src/main.js

```diff
@@ -53,6 +56,24 @@ function ymlContentToConfig(ymlContent) {
         throw new Error("Error parsing YAML: Google sheet id should be specified");
     }
 
+    if (configJson.sheet.name) {
+        config.sheetName = configJson.sheet.name
```

Comment:
There is a new parameter, but neither the README and the example config files are updated.